automatic/autoit.install/update.ps1

```diff
@@ -0,0 +1,46 @@
+import-module au
+
+$releases = "https://www.autoitscript.com/site/autoit/downloads"
+
+if ($MyInvocation.InvocationName -ne '.') { # run the update only if script is not sourced
+    function global:au_BeforeUpdate {
+        Remove-Item "$PSScriptRoot\tools\*.exe"
+
+        $filePath = "$PSScriptRoot\tools\$($Latest.FileName)"
+        Invoke-WebRequest $Latest.URL32 -OutFile $filePath
+
+        $Latest.ChecksumType = "sha256"
+        $Latest.Checksum = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath | ForEach-Object Hash
```

Comment:
damn, this also. I guess `Get-RemoteFiles` isn't really production ready yet then.
I hoped it would work, as Invoke-WebRequest is quite slow when downloading files.
oh well.